src/main/java/org/opensearch/knn/index/query/filtered/PlainFilteredKNNIterator.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.index.query.filtered;
+
+import org.apache.lucene.index.BinaryDocValues;
+import org.apache.lucene.search.DocIdSetIterator;
+import org.opensearch.knn.index.SpaceType;
+
+import java.io.IOException;
+
+/**
+ * Basic implementation of FilteredKNNIterator which iterate all doc IDs in filterIdsArray
+ */
+public class PlainFilteredKNNIterator<T extends KNNQueryVector> extends FilteredKNNIterator<T> {
```

Comment:
IMHO `Plain` prefix isn't very informative, for doc collectors core OS uses `Simple` (e.g. [here](https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/search/query/TopDocsCollectorContext.java#L341)), maybe we can follow the same here

Reply:
ack